src/server.cpp

```diff
@@ -148,8 +148,8 @@ Server::Server(
 		const std::string &path_world,
 		const SubgameSpec &gamespec,
 		bool simple_singleplayer_mode,
-		bool ipv6
-	):
+		bool ipv6,
```

Comment:
Hmmm, instead of continuously adding more boolean parameters, why not combine these into a set of "Server Startup" flags?